src/transforms/util/pull-request-link-generator.ts

```diff
@@ -1,11 +1,17 @@
 /**
  * Generates a create pull request link for GH in the format [URL]/compare/[name]?title=[...keys]-[name]
  * This format allows the use of title query param to set the PR name
+ * Max length: 2000
  */
 export const generateCreatePullRequestUrl = (baseUrl: string, name: string, issueKeys: string[] = []) => {
 	const keys = issueKeys?.length ? `${issueKeys.join(" ")} - ` : "";
 	const title = encodeURIComponent(keys + name);
 	const branchName = encodeURIComponent(name);
 	const url = `${baseUrl}/compare/${branchName}?title=${title}&quick_pull=1`;
+
+	if (url.length > 2000) {
+		return undefined;
```

Comment:
Returning undefined here will remove the `createPullRequestUrl` from the pullrequest payload

Reply:
Instead of returning undefined could we truncate the title?
(this is assuming the title that gets super duper long)

Reply:
If we truncate it won't be a valid URL though, so I guess it's better to not send any URL instead of sending a broken one

Reply:
Yeah, @gxueatlassian has a good point, if the currently url with the title is too long, we should be able to use the url without the title querystring and should be way under 2000 character as a fallback, but it means the user need to input the title manually, which I think is still a better experience than not having that create pr link :)